src/aiortc/rtcpeerconnection.py

```diff
@@ -1272,9 +1274,6 @@ def __validate_description(
             if not media.ice.usernameFragment or not media.ice.password:
                 raise ValueError("ICE username fragment or password is missing")
 
-            # check DTLS role is allowed
-            if description.type == "offer" and media.dtls.role != "auto":
```

Comment:
Do we need to also remove raising error in `__validate_description` when peer's role in peer's offer is not `actpass`? https://github.com/aiortc/aiortc/blob/e9c13eab915ddc27f365356ed9ded0585b9a1bb7/src/aiortc/rtcpeerconnection.py#L1277